Cargo.toml

```diff
@@ -7,8 +7,9 @@ authors = ["jrconlin <jconlin+git@mozilla.com>"]
 
 [dependencies]
 failure="~0.1"
-mysql="~12.2"
-rand="~0.4"
+futures="~0.2"
```

Comment:
Alex recommended to not use futures 0.2 as its an intermediary crate and breaks from 0.1 and will break again with 0.3.